src/Core/App.xaml.cs

```diff
@@ -713,6 +722,16 @@ public void Navigate(NavigationTarget navTarget, INavigationParams navParams)
             // If we are in background we add the Navigation Actions to a queue to execute when the app resumes.
             // Links: https://github.com/dotnet/maui/issues/11501 and https://bitwarden.atlassian.net/wiki/spaces/NMME/pages/664862722/MainPage+Assignments+not+working+on+Android+on+Background+or+App+resume
 #if ANDROID
+            var userVerificationMediatorService = ServiceContainer.Resolve<IFido2MakeCredentialConfirmationUserInterface>();
```

Comment:
⛏️ Having moved this to a `LazyResolve` field before makes it available to reuse it here without resolving again.